### PR TITLE
Fix yarn patch

### PR DESCRIPTION
**Pull request by jlandure**

### CODE REVIEW

File: onbuild/yarn/Dockerfile [hunk 1]
```diff
@@ -2,8 +2,14 @@ FROM zenika/alpine-node
 
 RUN npm install -g yarn
 
-ONBUILD COPY package.json /usr/src/app/
-ONBUILD RUN yarn 
+RUN adduser -D foo
```
Comment (fabienjuif) [inline onbuild/yarn/Dockerfile:7]: Add in comment why you do a foo user.

**Comment by fabienjuif** [comment]
Maybe you can add `VOLUME` entry to the yarn cache ?

**Comment by jlandure** [comment]
I don't want any default volume. The purpose of this image is to, by default, host all the dependencies without a volume.
But we can add one with runtime options.

**Comment by fabienjuif** [comment]
ok

**Review by fabienjuif — APPROVED** [review]

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**